lms/djangoapps/instructor/views/masters_gradebook_api.py

```diff
@@ -0,0 +1,128 @@
+"""
+Grade book view for instructor and pagination work (for grade book)
+which is currently use by ccx and instructor apps.
+"""
+import math
+
+from django.contrib.auth.models import User
+from django.db import transaction
+from django.views.decorators.cache import cache_control
+from opaque_keys.edx.keys import CourseKey
+
+from edxmako.shortcuts import render_to_response
+from lms.djangoapps.courseware.courses import get_course_with_access
+from lms.djangoapps.grades.course_grade_factory import CourseGradeFactory
+from lms.djangoapps.instructor.views.api import require_level
+from xmodule.modulestore.django import modulestore
+
+# Grade book: max students per page
+MAX_STUDENTS_PER_PAGE_GRADE_BOOK = 20
```

Comment:
Deleting the 2 functions below means we can get rid of this constant.  We'll probably also be able to remove many of the import statements.  Either pylint or pycodestyle should help tell you which ones aren't used after deleting the functions.